src/App.js

```diff
@@ -1,44 +1,94 @@
 import React, { Component } from 'react';
 import { View, StyleSheet, Platform } from 'react-native';
-import { NewsDetails, NewsList } from './screens';
+import { ScreenNewsDetails, ScreenNewsList, ScreenWebArticle } from './screens';
```

Comment:
Maybe it would be better to use convention `nameScreen`, for example `NewsListScreen`